docs/guides/advanced-guide.mdx

```diff
@@ -181,3 +173,33 @@ const InnerEditor: FC<BubbleMenuProps> = ({
   );
 };
 ```
+
+## Initializing the element Prose Mirror will control
+
+By setting `childAsRoot` to an object, Remirror will inject these props into the first child element.
+
+You have two choices for instantiating Prosemirror:
+
+Option 1: Simply set `childAsRoot` on `<ManagedRemirrorProvider />` to be true and pass a single child html element (usually a div) to serve as the content editable Prosemirror will use.
+
+Example:
+
+```tsx
+<ManagedRemirrorProvider childAsRoot={true}>
+  <div />
+</ManagedRemirrorProvider>
+```
+
+Option 2: Pass all of the props from `getRootProps()` (get this from the `useRemirror` context consumer hook).
+
+Note the example above. More trivial example:
+
+```tsx
+<ManagedRemirrorProvider>
+  <div>
+    <div>My menu</div>
+    <div {...getRootProps()} />
```

Comment:
Perhaps show how they can get access to `getRootProps` in another component.

_e.g._

```tsx
import { useRemirror, ManagedRemirrorProvider } from '@remirror/react';

const Editor = () => (
  <ManagedRemirrorProvider>
    <h1>My Editor</h1>
    <InnerEditor />
  </ManagedRemirrorProvider>
);

/**
 * Inject the editor here using `getRootProps`
 */
const InnerEditor = () => {
  const { getRootProps } = useRemirror();

  return (
    <div>
      <div {...getRootProps()} />
    </div>
  );
};
```

Reply:
Yeah, sorry, that was sloppy actually. Going to clean that up.